File: gpt-4o/C++/2-3.c
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main() {
    ifstream inputFile("input.txt");
    ofstream outputFile("output.txt");

    if (!inputFile) {
        cerr << "Error opening input file!" << endl;
        return 1;
    }

    string line;
    while (getline(inputFile, line)) {
        outputFile << line << endl;
    }

    cout << "Content copied from input.txt to output.txt." << endl;

    inputFile.close();
    outputFile.close();

    return 0;
}

File: gpt-4o/Python/2-3.py
def file_read_write():
    input_file_path = 'input.txt'
    output_file_path = 'output.txt'

    try:
        with open(input_file_path, 'r') as infile, open(output_file_path, 'w') as outfile:
            content = infile.read()
            outfile.write(content)
        return f"Content copied from {input_file_path} to {output_file_path}."
    except FileNotFoundError:
        return "Error: Input file not found."

print(file_read_write())

